### PR TITLE
Sync k8s migtraion to buildkite

### PR DESCRIPTION
**Pull request by pazone**
## What does this PR do?
MIgrates `Sync k8s` step to buildkite

### CODE REVIEW

**Review by dliappis — APPROVED** [review]
LGTM

**Comment by pazone** [comment]
The PR build is affected by https://github.com/elastic/elastic-agent/pull/3779

File: .buildkite/scripts/install-gh.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+then
+    set +e
+    echo "Found GH. Checking version.."
+    FOUND_GH_VERSION=$(gh --version 2>&1 >/dev/null | awk '{print $3}')
+    if [ "$FOUND_GH_VERSION" == "$GH_VERSION" ]
+    then
+        echo "GH Versions match: $GH_VERSION. No need to install gh. Exiting."
+        exit 0
+    else 
+        echo "GH VErsion mismatch. Desired version: $GH_VERSION, found version: $FOUND_GH_VERSION. Installing new version."
```
Comment (pchila) [inline .buildkite/scripts/install-gh.sh:25]:
```suggestion
        echo "GH Version mismatch. Desired version: $GH_VERSION, found version: $FOUND_GH_VERSION. Installing new version."    
```

**Review by pchila — APPROVED** [review]
A small typo in a log, looks good otherwise

**Comment by pchila** [comment]
buildkite test this

**Pull request merged**